chemprop/uncertainty/evaluator.py

```diff
@@ -72,39 +96,55 @@ class ExpectedNormalizedErrorEvaluator(RegressionEvaluator):
     and comparing the average predicted variance of the clusters against the RMSE of the cluster. [1]_
 
     .. math::
-        ENCE = \frac{1}{K} \sum_{i=1}^{K} \frac{|\text{mVAR}(i) - RMSE(i)|}{\text{mVAR}(i)}
+        \text{ENCE} = \frac{1}{N} \sum_{i=1}^{N} \frac{|\mathrm{RMV}_i - RMSE_i|}{\mathrm{mVAR}_i}
 
-    where :math:`mVAR(i)` is the root of the mean uncertainty over the :math:`i`-th bin and :math:`RMSE(i)`
+    where :math:`N` is the number of bins, :math:`RMV_i` is the root of the mean uncertainty over the :math:`i`-th bin and :math:`RMSE_i`
     is the root mean square error over the :math:`i`-th bin.This discrepancy is further normalized by the
-    uncertainty overthe bin, :math:`mVAR(i)`, because the error is expected to be naturally higher as the uncertainty increases.
+    uncertainty overthe bin, :math:`RMV_i`, because the error is expected to be naturally higher as the uncertainty increases.
```

Comment:
```suggestion
    uncertainty over the bin, :math:`\mathrm{RMV}_i`, because the error is expected to be naturally higher as the uncertainty increases.
```